framework/components/ecl-accordions/package.json

```diff
@@ -0,0 +1,25 @@
+{
+  "name": "@ec-europa/ecl-accordions",
+  "author": "European Commission",
+  "license": "EUPL-1.1",
+  "version": "0.1.0",
+  "description": "ECL Accordions",
+  "main": "_ecl-accordions.scss",
```

Comment:
in other components we didn't put "ecl-" as prefix for sass file. I don't really know the reason, but we should keep it consistant.